images/compose/compose.folders.yml

```diff
@@ -56,14 +93,18 @@ services:
         condition: service_healthy
       postgres:
         condition: service_healthy
+      migration_service:
+        condition: service_completed_successfully
+      set_init_password_service:
+        condition: service_completed_successfully
```

Comment:
This is not a technical dependency. Pulp can run just fine without that password. I think we should drop these lines.

In case of a configured SSO solution, you can still use pulp without admin.

Reply:
I vote for leave it as it is now. Or at least leave it as comment.
It would make easier for new users to find how to change the password without using `podman exec ...`.

Reply:
I'm not saying we should remove the service container. We just don't need it as a requirement on the api pod.
It will still start up anyway unless you call podman-compose in a special way.